Formula/zchunk.rb

```diff
@@ -0,0 +1,29 @@
+class Zchunk < Formula
+  desc "Compressed file format for efficient deltas"
+  homepage "https://github.com/zchunk/zchunk"
+  url "https://github.com/zchunk/zchunk/archive/refs/tags/1.3.0.tar.gz"
+  sha256 "5563baa254b256e30e1fea87f94f53a5cf63a074898644f3f7ae5b58f446db03"
+  license "BSD-2-Clause"
+  head "https://github.com/zchunk/zchunk.git", branch: "main"
+
+  depends_on "argp-standalone" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@1.1"
```

Comment:
Can we use `openssl@3`?

Reply:
My understanding was that we don't want to mix `openssl@1.1` and `openssl@3` in dependency trees. This formula also depends on `curl` (on Linux), which depends on `openssl@1.1`.

Reply:
Well, you have a dependency that isn't available on Linux, so that's not really an issue at the moment.

Reply:
Not really that keen on adding a new formula that depends on `openssl@1.1` until we're able to migrate most dependents to `openssl@3`, tbh. But if you're keen on having this in Homebrew, I wont object.

Reply:
No worries, I'll just put this in a tap for now.